src/main/resources/META-INF/plugin.xml

```diff
@@ -1,6 +1,6 @@
 <idea-plugin>
   <id>com.redhat.devtools.intellij.tekton</id>
-  <name>Tekton Pipelines by Red Hat</name>
+  <name>Tekton Pipelines by Red Hat (Deprecated)</name>
   <version>1.0</version>
   <vendor email="developers@redhat.com" url="https://www.redhat.com">Red-Hat</vendor>
```

Comment:
```suggestion
  <vendor email="devtools-team@redhat.com" url="https://www.redhat.com">Red-Hat</vendor>
```

Reply:
I think having the devtools-team email is more useful than having developers@redhat.com (as this thing does not exist)